librz/core/rop.c

```diff
@@ -538,7 +538,7 @@ static void analyze_gadget(RzCore *core, RzCoreAsmHit /*<RzCoreAsmHit *>*/ *hit,
 }
 
 void print_rop_gadget_info(RzCore *core, RzRopGadgetInfo *gadget_info) {
-	rz_cons_printf("Gadget 0x%" PFMT64x "\n", gadget_info->address);
+	rz_cons_printf("\nGadget 0x%" PFMT64x "\n", gadget_info->address);
```

Comment:
why the additional newline ?

Reply:
Separator between gadget output.
`\Rg` output:
```
Gadget 0x401004
Stack change: 0x8
Changed registers: rax rsp 
Register dependencies:
Var write: rax 128 140
Var write: rax 128 140

Gadget 0x401005
Stack change: 0x8
Changed registers: rax rsp 
Register dependencies:
Memory Read: rax 0
```

Reply:
ok, it makes sense.